spec/dummy/app/assets/javascripts/cable.js

```diff
@@ -0,0 +1,13 @@
+// Action Cable provides the framework to deal with WebSockets in Rails.
+// You can generate new channels where WebSocket features live using the `rails generate channel` command.
+//
+//= require action_cable
+//= require_self
+//= require_tree ./channels
+
+(function() {
+  this.App || (this.App = {});
+
+  App.cable = ActionCable.createConsumer();
+
+}).call(this);
```

Comment:
Would it be worth deleting unused generated cruft like this? (assuming it's unused…)
Same question for all the empty unused base classes like `ApplicationJob`.
Or it is simpler to just leave whatever Rails decided to generate?